tests/sqllogictests/suites/query/functions/02_0074_function_map.test

```diff
@@ -137,3 +137,51 @@ SELECT MAP_CAT(
 
 statement ok
 DROP DATABASE map_func_test
+
+# Test map_contains_key function
+statement ok
+CREATE DATABASE IF NOT EXISTS map_contains_key_test
```

Comment:
use db `map_func_test`, we don't need add a new database for this function test.